{
  "backup_data_uid": "b46044e1ac55e063",
  "backup_module_uid": "707ccdfe444cafac",
  "backup_module_uoa": "platform",
  "control": {
    "engine": "CK",
    "iso_datetime": "2020-11-14T21:43:51.541597",
    "version": [
      "1",
      "17",
      "0",
      "1"
    ]
  },
  "data_name": "XIAOMI MI 4I"
}
